es6-test/unit/merger.js

```diff
@@ -123,93 +124,79 @@ describe('merging feature streams to layers', function() {
       .pipe(merger)
       .on('end', (layers) => {
 
-        expect(layers.length).to.equal(1);
-
-        var [layer] = layers;
+        expect(layers.length).to.equal(2);
```

Comment:
Probably I'm misunderstanding what 'default crs' is, but I was reading it as a group of points that share the same crs, in which case, why are they on separate layers?